README.md

```diff
@@ -72,10 +73,10 @@ This repository is divided in 5 main packages:
 
 ### Installation
 
-Alexandria is a collection of [Scarb] packages, which can be installed by adding the following line to your `Scarb.toml`:
+Alexandria is a collection of cairo utilities, which can be installed by adding the following line to your `Scarb.toml`:
 ```toml
 [dependencies]
-alexandria_math = { git = "https://github.com/keep-starknet-strange/alexandria.git" }
+alexandria = { git = "https://github.com/keep-starknet-strange/alexandria.git" }
 ```
```

Comment:
Could you put here an example on how it should be imported in a Cairo file?
I know it wasn't there before but this could be useful 😄

Reply:
Yes ofc :)